horace_core/sqw/@sqw/generate_cube_sqw.m

```diff
@@ -0,0 +1,65 @@
+function out = generate_cube_sqw(shape)
+    % Generate a cube of SQW data FOR TESTING CUT/SYMMETRISE
+    % The data will be invalid in most circumstances
+    out = sqw();
+
+    if isscalar(shape) && isnumeric(shape)
+        ndim = 4;
+        npix = shape^ndim;
+        coords = zeros(9, npix);
+
+        if mod(shape, 2) == 0
+            col = [-shape/2 + 0.5:shape/2 - 0.5];
+        else
+            col = [-floor(shape/2):floor(shape/2)];
```

Comment:
this is a bit opaque. If shape is restricted to integers, as in the PR main comment, then floor(shape/2) is equivalent to (shape+1)/2 which would be easier to think about for maintenance purposes. It would be useful to comment this at the top of the function and test for it.

if in fact shape is any real value, then again comment at top of function. But is that what you want?